lib/puppet/module.rb

```diff
@@ -353,8 +364,27 @@ def ==(other)
     self.environment == other.environment
   end
 
+  def initialize_i18n
+    locales_path = File.absolute_path('locales', path)
+    config_path = File.absolute_path('config.yaml', locales_path)
+    module_name = @forge_name.gsub("/","-") if @forge_name
+
+    return nil if i18n_initialized? module_name
```

Comment:
it's not designed to fail, it just warns that there won't be any translations. i didn't think it was worth blocking a puppet run because translations aren't running. it's not affecting system configuration or anything, just the language of the log output, which will be obvious.

also, i like your version of the function a lot. thanks for the review!